# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/test/crashtests/crashtests.list
# Commit: fd44d547e093
# Full Hash: fd44d547e0931ddcb25e9dde2691ad9e99ec2e05
# Author: alwu <alwu@mozilla.com>
# Date: 2023-10-26 09:13:45
# Description:
#   Bug 1860840 - return error for invalid hvcc config when converting sample. r=media-playback-reviewers,padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D191794
# ==============================================================================

diff -r 691148c77012 -r fd44d547e093 dom/media/test/crashtests/crashtests.list
--- a/dom/media/test/crashtests/crashtests.list	Wed Oct 25 21:14:33 2023 +0300
+++ b/dom/media/test/crashtests/crashtests.list	Wed Oct 25 17:10:46 2023 +0000
@@ -176,3 +176,4 @@
 load 1850453.html
 test-pref(media.wmf.hevc.enabled,1) load 1859384.mp4
 test-pref(media.wmf.hevc.enabled,1) load 1859600.mp4
+test-pref(media.wmf.hevc.enabled,1) load 1860840.mp4
